setup.py

```diff
@@ -55,7 +55,7 @@ def _get_long_description():
     install_requires=[
         'click>=6.7,<9',
         'pydantic>=1,<2',
-        'redis>=4.2,<5',
+        'redis>=4.2,<6',
```

Comment:
Suppor for verstion < 5 is broken now, you can use aliases.

Reply:
Made changes backwards compatible